### DISCUSSION TITLE
阿里云目录包含 / 就会导致failed get objs:错误

### DISCUSSION THREAD

**Discussion by mixpplus** [AlistGo/alist#2071] [General]
#### 在实际阿里云分享链接中是存在文件的，如下
![image](https://user-images.githubusercontent.com/47939006/197123316-252601fa-f7ce-41c6-a49d-e013bf6f81d7.png)
#### 但是在目录名称中存在`/`字符，导致在alist服务中查找出错，如下：
![image](https://user-images.githubusercontent.com/47939006/197123629-a7eb1032-782c-4796-a5d4-179a591fa140.png)
#### 由于是分享连接，无权限更改文件夹名称，希望大佬更改一下此部分的实现逻辑，万谢！！！

**Comment by anwen-anyi** [comment]
应该不行了.. **"/"** 在Alist这里是路径点，以下的都不行吧...
 - 不能包含?，这表示一个parameters参数
 - 不能包含#，这表示一个anchor参数
 - 不能包含/，这用于定位路径
 - 不能包含\，错误的转义字符会导致程序陷入死循环 具体可参考https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL

**Reply by mixpplus** [reply]
其实我觉得阿里云盘网页端处理路径节点的方式可以作为参考，他们使用`>`来截断的，这样的话只要文件夹不包含`>`符号，其他任何符号都可以正常工作！
